assets/js/modules/search-console/components/common/AnalyticsStats.js

```diff
@@ -32,6 +32,38 @@ import { extractAnalytics4DashboardData } from '../../../analytics-4/utils';
 import GoogleChart from '../../../../components/GoogleChart';
 const { useSelect } = Data;
 
+function extractChartData(
```

Comment:
@hussain-t, please, add jsdoc for this function.

Reply:
Ah, thanks for catching it 👍